onnxruntime/python/tools/transformers/bert_perf_test.py

```diff
@@ -159,9 +209,14 @@ def run_one_test(model_setting, test_setting, perf_results, all_inputs, intra_op
     print("Running test:", key)
 
     all_latency_list = []
-    for i in range(test_setting.test_times):
-        results, latency_list = onnxruntime_inference(session, all_inputs, output_names)
-        all_latency_list.extend(latency_list)
+    if test_setting.use_io_binding:
```

Comment:
Nit: if else could be inside the for loop to reduce code duplication.

Reply:
good point, perhaps next time whoever touch the code, we'll remind it.